### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by PhantumPhysics** [IAmPicard/StarTrekTimelinesSpreadsheet#247]
**Describe the bug**
During a voyage the estimated chance to reach next dilema shows 0% although the estimated duration is very close to the time to new dilema

**To Reproduce**
Just look at the voyage status for an ongoing voyage close to the end.

**Expected behavior**
I would expect there to be a small random chance

**Screenshots**
Here is an example
![image](https://user-images.githubusercontent.com/42848368/49633492-315f2e00-fa02-11e8-8245-8e10bbb5dd18.png)

**Additional context**
This may be working as intended since I know the random spread drops after crew no longer can pass skill checks.  I have noticed this a few times that expected duration seems to be at odds with the chance of the next dilema later in the voyage.  I thought I would report this to see if there was a problem somewhere in the calculations or the display.

Tool version: **0.8.2-web**
Operating system: **Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/71.0.3578.80 Safari/537.36**

**Comment by jdarcy2012** [comment]
If you ask me, getting to the point of needing the actual probability in those last few minutes before the next dilemma is a total edge case.  Given that it relies on the voyage length algo to even establish whether or not this probability calculation is necessary (will the voyage get within 1-5 minutes of the next dilemma?), which is not entirely reliable, this feature should probably be removed.  If the voyage is estimated to go close to the next dilemma but not make it convincingly, the user should probably be paying attention for the last 10 minutes or so.

Recommendation: Deprecate probability feature.

**Comment by IAmPicard** [comment]
Yes, this mostly tells you if there is any chance at all to reach next dilemma; I should probably switch it into a tri-state-flag (no chance / maybe / absolutely).

The actual percentage will vary wildly near the end as the hazards are generally failing at a much higher rate than average.
